src/main/java/org/gridsuite/directory/server/DirectoryController.java

```diff
@@ -37,104 +37,98 @@ public DirectoryController(DirectoryService service) {
     }
 
-    @PutMapping(value = "/directories/{elementUuid}/rename/{newElementName}")
+    @PutMapping(value = "/elements/{elementUuid}")
     @Operation(summary = "Rename element/directory")
     @ApiResponses(value = {
         @ApiResponse(responseCode = "200", description = "Element/directory was successfully renamed"),
         @ApiResponse(responseCode = "404", description = "The element was not found"),
         @ApiResponse(responseCode = "403", description = "Not authorized to rename this element")
     })
     public ResponseEntity<Mono<Void>> renameElement(@PathVariable("elementUuid") UUID elementUuid,
-                                                    @PathVariable("newElementName") String newElementName,
+                                                    @RequestParam("name") String newElementName,
                                                     @RequestHeader("userId") String userId) {
         return ResponseEntity.ok().body(service.renameElement(elementUuid, newElementName, userId));
     }
 
-    @PutMapping(value = "/directories/{elementUuid}/rights", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @PutMapping(value = "/elements/{elementUuid}", consumes = MediaType.APPLICATION_JSON_VALUE)
     @Operation(summary = "Modify element/directory's access rights")
     @ApiResponses(value = {
         @ApiResponse(responseCode = "200", description = "Element/directory was successfully updated"),
         @ApiResponse(responseCode = "404", description = "The element was not found"),
         @ApiResponse(responseCode = "403", description = "Not authorized to update this element access rights")
     })
     public ResponseEntity<Mono<Void>> setAccessRights(@PathVariable("elementUuid") UUID elementUuid,
-                                                      @RequestBody boolean isPrivate,
+                                                      @RequestParam("private") boolean isPrivate,
                                                       @RequestHeader("userId") String userId) {
         return ResponseEntity.ok().body(service.setAccessRights(elementUuid, isPrivate, userId));
     }
 
-    @DeleteMapping(value = "/directories/{elementUuid}")
+    @DeleteMapping(value = "/elements/{elementUuid}")
     @Operation(summary = "Remove directory/element")
     @ApiResponses(@ApiResponse(responseCode = "200", description = "Directory/element was successfully removed"))
     public ResponseEntity<Mono<Void>> deleteElement(@PathVariable("elementUuid") UUID elementUuid,
                                                     @RequestHeader("userId") String userId) {
         return ResponseEntity.ok().body(service.deleteElement(elementUuid, userId));
     }
 
-    @RequestMapping(method = RequestMethod.HEAD, value = "/directories/{directoryUuid}/{elementName}")
-    @Operation(summary = "Check if the element exists")
+    @RequestMapping(method = RequestMethod.HEAD, value = "/elements/{elementUuid}/elements/{elementName}")
```

Comment:
/elements/{elementUuid} to check an element existence including a directory

Reply:
This endpoint test if an element exist in a directory and not the esistence of an element.
Old path :
 `@RequestMapping(method = RequestMethod.HEAD, value = "/directories/{directoryUuid}/{elementName}")`

Reply:
`@RequestMapping(method = RequestMethod.HEAD, value = "/directories/{directoryUuid}/elements/{elementName}") @Operation(summary = "Check if an element exists in a directory")`

Reply:
ok!